README.md

```diff
@@ -2,3 +2,16 @@
 Face/car recognition API in .NET core using layered architecture with monolithic deployment model.
 * [Mobile application](https://github.com/tozaicevas/epicenter-mobile)
 * [Web application](https://github.com/UndeadRat22/EpicenterWebapp)
+
+Endpoints:
```

Comment:
could use ## here to split, done that for you, otherwise great job and thanks :)))